Khan/khan-linter
# Add eslint-plugin-react-native

We have a lot of unused styles in webapp but it's hard to tell which ones
are unused.  This new plugin contains a lint ruled called no-unused-styles
which does exactly what we want.  Even though we're not using react-native
in webapp, the API for StyleSheet matches aphrodite's StyleSheet so this
rule will work for our code.  I'll post a separate diff to enable this rule
in webapp and remove the unused styles.

This change was generated by running `yarn add eslint-plugin-react-native`.

Test Plan:
- enable this rule in webapp
- run "ka-lint javascript" in webapp
- see it report all of the unused styles
